Add missing const in deleted copy constructor (from b6e8fc10f)

This fails only with g++<9, probably because of P0641R2 which was added in
g++-9.

diff --git codels/codels.hh codels/codels.hh
index 257d782..151b9a5 100644
--- codels/codels.hh
+++ codels/codels.hh
@@ -40,7 +40,7 @@
 
 struct camera_pub {
   camera_pub(const std::string &bank) : bank(bank) {}
-  camera_pub(camera_pub &) = delete;
+  camera_pub(const camera_pub &) = delete;
   void operator=(const camera_pub &) = delete;
 
   camera_pub *src(uint8_t index, const std::string &slot);
